org.technic93.e2designer.yaml

```
@@ -0,0 +1,20 @@
+app-id: "org.technic93.e2designer"
+runtime: org.kde.Platform
+runtime-version: '5.12'
```

```suggestion
runtime-version: '5.12'
```
Have you tried bumping this to 5.13?